src/protocol/SFSCommunication.h

```diff
@@ -1239,7 +1239,7 @@ enum class SugidClearMode {
 /// msgid:32 status:8
 
 //0x01E6
-#define SAU_CLTOMA_REGISTER_CONFIG (1000U + 485U)
+#define SAU_CLTOMA_REGISTER_CONFIG (1000U + 486U)
```

Comment:
This also looks nice. (Makes equal constant and comment, though it is probably not necessary for the fix you are doing).

Reply:
The bug was related to this commit https://github.com/leil-io/saunafs/commit/488ee3e0228efd6ae39e0865ca6d48574606aeff. At first I thought the problem was here, but I was wrong. I still kept the fix in though, since it's still kinda related.

Reply:
I've updated the PR description